### ISSUE TITLE
Create global.js file

### ISSUE THREAD

**Issue by noahklein00** [joe-studer-18/mst-ka-site#31]
Create a global.js file that stores and displays regularly updated/important information across the website.

**Comment by noahklein00** [comment]
I got this working, but I need to know everywhere we want to add the global.js replacement for. Also, the <p> tag that I have the javascript replace the text in will not go inline unless I put the css styling directly into the <p> tag like <p style="">. If someone knows where I can add this to one of the css files that would be great but when I tried it wasn't working.

**Comment by joe-studer-18** [comment]
the only one I can think of right now is to update the president's phone number in the officer contact page & footer, as well as the alumnus advisor's email in the footer pages. I think most of the stuff in the footer should be in this file, would you agree (at least the more dynamic ones).

Other than that, we can all kinda update files as we go/catch stuff in PRs.

As for the "p" tag issue, that maaay be with how we're modifying the text, I think there's several possible ways to do it (maybe one will protect the css of the element?). Is this issue only on "p" elements?

**Comment by noahklein00** [comment]
The issue with the "p" elements is somewhat similar to if I tried to put it into a "span" as well. I just can't find where to overwrite the css that is affecting these tags by default. If I put it in a "span" tag, the color of the numbers turns red according to one of the css properties that the template we took from had in place. If I put it in just a "div" tag, it newlines the phone numbers and again, I cannot figure out where to redo this. The "p" tag with the style attribute allows me to keep the text white and inline.
